CHANGELOG.md

```diff
@@ -16,9 +16,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Bug fixes
 
 ### Dependencies
+- Update datadog-api-client dependency from 2.9.0 to 2.10.0 ([#80](https://github.com/personio/datadog-synthetic-test-support/pull/80))
```

Comment:
is this here by mistake?

build.gradle.kts diff contains only kotlin-stdlib version upgrade 🤔